--- boost_1_56_0/boost/config/compiler/clang.hpp.old	2014-11-12 18:33:03.000000000 -0800
+++ boost_1_56_0/boost/config/compiler/clang.hpp	2014-11-12 18:36:43.000000000 -0800
@@ -100,9 +100,10 @@
 #  define BOOST_NO_CXX11_DEFAULTED_FUNCTIONS
 #endif
 
-#if !__has_feature(cxx_deleted_functions)
+// Caused "<type_traits>: file not found" error building mezzanine-screencast with libstdc++ and with greenhouse on osx107
+//#if !__has_feature(cxx_deleted_functions)
 #  define BOOST_NO_CXX11_DELETED_FUNCTIONS
-#endif
+//#endif
 
 #if !__has_feature(cxx_explicit_conversions)
 #  define BOOST_NO_CXX11_EXPLICIT_CONVERSION_OPERATORS
@@ -140,17 +141,19 @@
 #  define BOOST_NO_CXX11_RAW_LITERALS
 #endif
 
-#if !__has_feature(cxx_reference_qualified_functions)
+// Breaks sandbox compilation with libstdc++
+//#if !__has_feature(cxx_reference_qualified_functions)
 #  define BOOST_NO_CXX11_REF_QUALIFIERS
-#endif
+//#endif
 
 #if !__has_feature(cxx_generalized_initializers)
 #  define BOOST_NO_CXX11_UNIFIED_INITIALIZATION_SYNTAX
 #endif
 
-#if !__has_feature(cxx_rvalue_references)
+// Caused problems building mezzanine-screencast with libstdc++
+//#if !__has_feature(cxx_rvalue_references)
 #  define BOOST_NO_CXX11_RVALUE_REFERENCES
-#endif
+//#endif
 
 #if !__has_feature(cxx_strong_enums)
 #  define BOOST_NO_CXX11_SCOPED_ENUMS
